ejb3/src/main/java/org/jboss/as/ejb3/remote/AssociationImpl.java

```diff
@@ -247,19 +248,23 @@ public SecurityIdentity getSecurityIdentity() {
     private void updateAffinities(InvocationRequest invocationRequest, Map<String, Object> attachments, EJBLocator<?> ejbLocator, ComponentView componentView) {
         Affinity legacyAffinity = null;
         Affinity weakAffinity = null;
+        Affinity strongAffinity = null;
         Affinity clusterAffinity = getClusterAffinity();
```

Comment:
Remove this line.  We need to discourage people from thinking that "cluster affinity" is a proper affinity type.  Affinity types are weak and strong, that's it.